### PR TITLE
add config introduction about nodes and sdk-demo

### PR DESCRIPTION
**Pull request by LucasLi1024**

### CODE REVIEW

**Comment by ghost** [comment]
Sider has detected **1** warning on analyzing the commit f181b54.

- ⚠️ [Goodcheck](https://sider.review/gh/repos/148607759/pulls/1463/logs/d4928bce-a87f-437f-8f16-0c950536d4ba)

If the errors persist even after [retrying](https://sider.review/gh/repos/148607759/pulls/1463#tools), the following actions may resolve them:
- Update tool dependencies
- Configure failed tools
- [Configure `sider.yml`](https://sider.review/gh/repos/148607759/pulls/1463#config)
- [Turn off unused tools](https://sider.review/gh/repos/148607759/settings)

If you still have problems, feel free to ask us via chat. 💬

---

You can [turn off such notifications](https://sider.review/gh/repos/148607759/settings#notifications) if unnecessary.

File: 3.x/zh_CN/docs/tutorial/air/multihost.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+**下载控制台**
+
+```bash
+# 创建操作目录
+mkdir -p ~/fisco && cd ~/fisco
+
+curl -#LO https://github.com/FISCO-BCOS/console/releases/download/v3.0.0-rc3/download_console.sh && bash download_console.sh
+
+# 若因为网络问题导致长时间无法下载，请尝试以下命令：
+curl -#LO  https://gitee.com/FISCO-BCOS/console/raw/master-2.0/tools/download_console.sh && bash download_console.sh
```
Comment (cyjseagull) [inline 3.x/zh_CN/docs/tutorial/air/multihost.md]:
```suggestion
curl -#LO  https://gitee.com/FISCO-BCOS/console/raw/master/tools/download_console.sh && bash download_console.sh
```

File: 3.x/zh_CN/docs/tutorial/air/multihost.md [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+mkdir -p ~/fisco && cd ~/fisco
+
+# 下载开发部署工具build_chain
+curl -#LO https://github.com/FISCO-BCOS/FISCO-BCOS/releases/download/v3.0.0-rc3/build_chain.sh && chmod u+x build_chain.sh
+```
+
+**生成多机节点配置**
+
+```bash
+bash build_chain.sh -l "196.168.0.1:1,196.168.0.2:1,196.168.0.3:1,196.168.0.4:1" -o 4nodes -e ./fisco-bcos
```
Comment (cyjseagull) [inline 3.x/zh_CN/docs/tutorial/air/multihost.md]: Supplementary steps for downloading fisco-bcos.

File: 3.x/zh_CN/docs/develop/stress_testing.md [hunk 3]
```diff
@@ -197,6 +231,14 @@ vim ～/fisco/nodes/127.0.0.1/node0/config.ini
     max_log_file_size=200
 ```
 
+若压测交易量比较大，则增加交易池数量限制。如压测的交易总量为500000，则`limit`要设置大于500000。默认为15000。否则节点会报`txpoolIsFull`错误。
+
+```ini
+[txpool]
+    ; size of the txpool, default is 15000
+    limit=15000
```
Comment (cyjseagull) [inline 3.x/zh_CN/docs/develop/stress_testing.md]:
```suggestion
    limit=1500000
```

File: 3.x/zh_CN/docs/develop/stress_testing.md [hunk 4]
```diff
@@ -167,19 +177,43 @@ tree -L 2 ～/fisco/nodes
...
+
+    consensus_timeout=3000
+    leader_period=1
+    node.0=18325b7816386b0986a00e1059219ffe898059cfc4b8496551c7262d09836e3901ef1ad68f3a72b311628292c535680800b31ec2772d1d240b658fcdd3a658af: 1
+    node.1=4fcf0ebaaa1e5646519142ff91b937618cfd3d4c9571071cb02dbab19a5da95b1d70dcc7b87620290826fb734f46e7bbf3dde9f6071cd31b16ca17301e3a6ae8: 1
+    node.2=05daa92decd4144b31332ee1356c1a7f68f83b88cbbe8d5e0d141a0598b4ec2d2199adea18ce0a311b3d11001ef1121a8500b5fb33e1bef268600789bae1650e: 1
+    node.3=027f25b1597d363babe412962b05995d56033636cb061737beeb7e9a6c811e19f1fccf763acc8271ba542eb4fe5d798e9b06ce0e28ef73285e7b86778ad879ca: 1
+```
+
+若是已经启动过节点了，再修改创世块配置的话，需要删除掉节点原先data目录下的数据，再重启节点。（因为节点启动后，将创世块配置写入到data目录下，在data目录已经有数据的时候，就不会再重新读取config.genesis的配置）。
```
Comment (cyjseagull) [inline 3.x/zh_CN/docs/develop/stress_testing.md]:
```suggestion
若是已经启动过节点了，再修改创世块配置需删除节点data目录下的数据，再重启节点。
```

File: 3.x/zh_CN/docs/develop/stress_testing.md [hunk 5]
```diff
@@ -167,19 +177,43 @@ tree -L 2 ～/fisco/nodes
...
+vim ～/fisco/nodes/127.0.0.1/node0/config.genesis
+```
+
+```ini
+[consensus]
+    consensus_type=pbft
+
+    ; 修改该数值为2000，默认1000
+    ; the max number of transactions of a block
+    block_tx_count_limit=1000
```
Comment (cyjseagull) [inline 3.x/zh_CN/docs/develop/stress_testing.md]:
```suggestion
    block_tx_count_limit=2000
```

File: 3.x/zh_CN/docs/tutorial/air/multihost.md (hunk 1 above)
Comment (LucasLi1024) [reply 3.x/zh_CN/docs/tutorial/air/multihost.md]: fixed

File: 3.x/zh_CN/docs/tutorial/air/multihost.md (hunk 2 above)
Comment (LucasLi1024) [reply 3.x/zh_CN/docs/tutorial/air/multihost.md]: has added Supplementary steps

File: 3.x/zh_CN/docs/develop/stress_testing.md (hunk 3 above)
Comment (LucasLi1024) [reply 3.x/zh_CN/docs/develop/stress_testing.md]: fixed

File: 3.x/zh_CN/docs/develop/stress_testing.md (hunk 4 above)
Comment (LucasLi1024) [reply 3.x/zh_CN/docs/develop/stress_testing.md]: fixed

File: 3.x/zh_CN/docs/develop/stress_testing.md (hunk 5 above)
Comment (LucasLi1024) [reply 3.x/zh_CN/docs/develop/stress_testing.md]: fixed

**Review by cyjseagull — APPROVED** [review]

**Pull request merged**